travis/test_measures.sh

```diff
@@ -6,6 +6,8 @@ if ! ./travis/rebuild_bundles.sh; then
 fi
 
 cd fhir-patient-generator
+git checkout ci-param-tool-rebase
+make all all-r4 CI_TOOL=true
```

Comment:
Do you actually need both the `all` and `all-r4` targets here? I think they do the same thing in FHIR Patient Generator.

Reply:
Also I just realized my comment could be confusing-- I believe that `make all` and `make all-r4` are aliases for the same makefile target in FHIR Patient Generator, is what I meant to say.